### PR TITLE
My first pull request!!!

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request I:
- added index.html and header
- added header in index.html
- I stylized the header

### CODE REVIEW

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @vladanvi99 ✋,

Amazing job so far, but you need to make a few changes so that your project can be approved but don't worry you are almost there!

## Required Changes ♻️

- [ ] Please add a short description in your PR to explain the change you made in your project you can refer to this [Link](https://microverse.zendesk.com/hc/en-us/articles/360039277914-12-2-2019-Professional-Github-repository-lesson) on how to create a professional repo
- [ ] Please work on your README to make it professional refer to this [Template](https://github.com/microverseinc/readme-template) 

### Cheers, and Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DeliceLydia** [comment]
Hello @vladanvi99,

I am sorry I invalidated your PR by mistake you can submit it again
so that It can be reviewed.

I apologize for the inconvenience!

**Comment by vladanvi99** [comment]
> 
> 
> Hello @vladanvi99,
> 
> I am sorry I invalidated your PR by mistake you can submit it again
> so that It can be reviewed.
> 
> I apologize for the inconvenience!

File: README.md [hunk 1]
```diff
@@ -4,6 +4,42 @@
...
+
+No Live Preview
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/vladanvi99)
+- Twitter: [@twitterhandle](https://twitter.com/vladanvi99)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/vladan-videnovi%C4%87-780bb11b2/)
```
Comment (jubaan) [inline README.md:26]: [OPTIONAL] 

- [ ] the `githubhandle` is just a placeholder within the template, feel free to change it with your own username.
[@vladanvi99](https://github.com/vladanvi99)

**Review by jubaan — APPROVED** [review]
Hi @vladanvi99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**